cuda-getting-started/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-cmake_minimum_required(VERSION 3.1)
-project(cis565_getting_started)
+cmake_minimum_required(VERSION 3.27)
+project(cis565_getting_started LANGUAGES CUDA CXX)
```

Comment:
I suggest that we add C here, just in case some libs that students wants to use are in C.

Reply:
@bdwhst Based on the base code we have CXX is fine. If students want to use pure C, then they can modify the CMake file.